build.gradle

```diff
@@ -69,66 +66,59 @@ subprojects {
-//              email = "{please_input_your_email}"
-//            }
-//          }
-          scm {
-            connection = "scm:git:git://github.com/naver/ngrinder.git"
-            developerConnection = "scm:git:ssh://github.com/naver/ngrinder.git"
-            url = "https://github.com/naver/ngrinder"
-          }
-        }
-      }
-    }
+  artifacts {
+    archives javadocJar, sourceJar
   }
 
-  bintray {
-    user =  project.hasProperty("bintrayUser") ? bintrayUser : ""
-    key =  project.hasProperty("bintrayKey") ? bintrayKey : ""
-    publications = ["nGrinerModules"]
-    publish = true
-    override = true
-
-    pkg {
-      repo = "ngrinder"
-      name = project.name
-      userOrg = "navercorp"
-      licenses = ["Apache-2.0"]
-      websiteUrl = "https://github.com/naver/ngrinder"
-      issueTrackerUrl = "https://github.com/naver/ngrinder/issues"
-      vcsUrl = "https://github.com/naver/ngrinder.git"
-      labels = ["ngrinder"]
-      publicDownloadNumbers = true
-      version {
-        name = project.version
-        released = new Date()
-        gpg {
-          sign = true
-          passphrase = project.hasProperty("bintrayGpgPassphrase") ? bintrayGpgPassphrase : ""
-        }
-      }
-    }
-  }
+// Please remove below comments if you need to deploy ngrinder binary to OSSRH.
+//
+//  signing {
+//    sign configurations.archives
+//  }
+//
+//  uploadArchives {
+//    repositories {
+//      mavenDeployer {
+//        beforeDeployment { deployment -> signing.signPom(deployment) }
+//
+//        repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+//          authentication(userName: ossrhUsername, password: ossrhPassword)
+//        }
+//
+//        snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
+//          authentication(userName: ossrhUsername, password: ossrhPassword)
+//        }
+//
+//        pom.project {
+//          name = "org.ngrinder:${project.name}"
+//          description = "${project.name} module"
+//          url = "https://github.com/naver/ngrinder"
+//
+//          scm {
+//            connection = "scm:git:git://github.com/naver/ngrinder.git"
+//            developerConnection = "scm:git:ssh://github.com/naver/ngrinder.git"
+//            url = "https://github.com/naver/ngrinder"
+//          }
+//
+//          licenses {
+//            license {
+//              name = "The Apache License, Version 2.0"
+//              url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+//            }
+//          }
+//
+////          Below field is required to publish to the Maven central. Please modify before publishing.
+////          developers {
+////            developer {
+////              id = "{please_input_your_id}"
+////              name = "{please_input_your_name}"
+////              email = "{please_input_your_email}"
+////            }
+////          }
+//
+//        }
+//      }
+//    }
+//  }
```

Comment:
Just remove out commented part.

Reply:
Yes, I changed it to be active only when all required properties exist.

https://github.com/naver/ngrinder/blob/786f6e15798160e5836d1447c9280fe014a381a9/build.gradle#L73

Reply:
```
//          Below field is required to publish to the Maven central. Please modify before publishing.
//          developers {
//            developer {
//              id = "{please_input_your_id}"
//              name = "{please_input_your_name}"
//              email = "{please_input_your_email}"
//            }
//          }
```
please fix this as well

Reply:
I just removed the comments.
Before deploying to OSSRH, this information must be modified to valid values.

https://github.com/naver/ngrinder/blob/6438df2174422f74b4c93e54549cc37cc5c3480d/build.gradle#L113